src/config-util/index.js

```diff
@@ -0,0 +1,6 @@
+'use strict'
+
+/* eslint node/exports-style: 0 */
```

Comment:
i prefer we don't override our ESLint config per-file basis like this. as this project grows, there'll be drawbacks in terms of maintainability

Reply:
I could do the following:
- `module.exports.whatever = whatever`.
- Not have an `index.js` file.
- Change eslint config.

If I have to _change_ something, probably would go with 3.

Reply:
> Let's do 1, please.

Ok.

> it seems module.export is what most packages are using?

Not sure. I usually use `module.exports` for single export and `exports.whatever` for multiple.

Reply:
@sudo-suhas is there any semantical difference between those 2 methods? I definitively opposed of using 2 different methods if they are equal semantically. We could modify the eslint config to enforce `export` pattern.

Reply:
Here's some relevant documentation - https://nodejs.org/api/modules.html#modules_the_module_wrapper and a blog post - https://medium.freecodecamp.org/node-js-module-exports-vs-exports-ec7e254d63ac.

So my understanding is before the file is `require`ed, exports are setup with something like this:
```js
exports = module.exports = {}
```

So `exports.whatever = whatever` is equivalent to `module.exports.whatever = whatever`. However, `exports = whatever` won't work.